packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -693,6 +700,21 @@ export default defineComponent({
         resourceType,
         ownerName: resource.owner[0].displayName
       })
+    },
+    getMatchingSpace(storageId) {
+      return this.spaces.find((space) => space.id === storageId)
+    },
+    getDefaultParentFolderName(resource) {
+      if (!this.hasSpaces) {
+        this.$gettext('All files and folders')
+      }
+      const matchingSpace = this.getMatchingSpace(resource.storageId)
+
+      if (matchingSpace && matchingSpace?.driveType === 'project') {
```

Comment:
```suggestion
      if (matchingSpace?.driveType === 'project') {
```

Reply:
matching space might be null, don't think optional chaining only will work here

Reply:
Isn't the optional chaining for such cases? I thought if `matchingSpace` is `null`, then you effectively have `if (null === 'project') {...}`.

Reply:
Let me check but I don’t think it works on null values, just on objects

Reply:
null?.foo?.bar shouldn’t work, but {}?.foo?.bar should

Reply:
you was right ;)